compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -33,6 +33,7 @@ i4125.scala
 i4167
 i4586.scala
 i4734
+i4758
```

Comment:
They slipped through the CI because before #5059 the CI didn't run `-Ytest-pickler` on pos and run tests..